src/celeritas/ext/GeantImporter.cc

```diff
@@ -250,6 +260,8 @@ fill_vec_import_scint_comp(MatPropGetter& get_property,
                             ImportUnits::time);
         if (comp)
         {
+            // TODO: should we error if the user defines some but not all
```

Comment:
Looking at our generator it seems like we need all, except for the rise time, that can be set to zero. Everything else must be non-zero to avoid either odd behaviors or straight up errors: if the fall time is not set we end up with a division by zero in `ScintillationGenerator::148`.

Reply:
It makes sense that we need it all, but since these properties aren't really validated by Geant4 perhaps we should throw an error saying "hey we expected all but found these". Or perhaps that should be done inside the scintillation generator itself from the import data.

Reply:
Yeah, we do validate the imported components in the scintillation params...